src/manifest.json

```diff
@@ -26,12 +26,6 @@
 
   // testing version allows loading unit test libraries from CDNs
   "content_security_policy": "default-src 'self'; img-src data:; style-src 'self' https://unpkg.com; script-src 'self' https://unpkg.com",
-  "icons": {
-    "16": "icons/icon-small-colored.svg",
-    "32": "icons/icon-small-colored.svg",
-    "48": "icons/icon-large.svg",
-    "96": "icons/icon-large.svg"
-  },
```

Comment:
This would also need to be removed from the other `manifest.json` files until #15 is merged, particularly `chromemanifest.json`.

Reply:
Oh I guess this was an error due to testing it in Chrome/ium. Thx.